OrbitGl/EventTrack.cpp

```diff
@@ -70,8 +70,8 @@ void EventTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick, PickingM
   const bool picking = picking_mode != PickingMode::kNone;
 
   ScopeLock lock(GEventTracer.GetEventBuffer().GetMutex());
-  std::map<uint64_t, CallstackEvent>& callstacks =
-      GEventTracer.GetEventBuffer().GetCallstacks()[thread_id_];
```

Comment:
Here's one thing I don't understand: This is one of the two lines that caused the insertion. Still, this inserts an empty map of callstack events. Since it's empty and there are no actual events, shouldn't the code that computes the duration of the capture (note: I'm not sure where that is) produce zero anyway, as there are still effectively not callstack events? Somehow I have the feeling that while this is one cause, the real cause could be in that other logic.

Reply:
It should now not insert an empty callstack anymore, but just return the static const empty callstack defined in the callee.

Reply:
But what is the code that adds an empty callstack? This specific line only adds an empty map.

Reply:
Discussed separately, fixed `HasEvent`.